test/README.md

```diff
@@ -155,6 +155,26 @@ import pdb; pdb.set_trace()
 anywhere in the test. You will then be able to inspect variables, as well as
 call methods that interact with the bitcoind nodes-under-test.
 
+If further introspection of the bitcoind instances themselves becomes
+necessary, this can be accomplished by first setting a pdb breakpoint
+at an appropriate location, running the test to that point, then using
+`gdb` to attach to the process and debug:
+
+```bash
```

Comment:
I think this can be more easily achieved by `cat`ing the `.pid` file in the data directory. Find the data directory:

```bash
2017-06-27 14:13:56.686000 TestFramework (INFO): Initializing test directory /tmp/user/1000/testo9vsdjo3
```

Then `cat` the `.pid` file:

```bash
cat /tmp/user/1000/testo9vsdjo3/node0/regtest/bitcoind.pid
gdb /home/example/bitcoind !$
```

(or if you really have to show off):

```bash
gdb /home/example/bitcoind $(cat /tmp/user/1000/testo9vsdjo3/node0/regtest/bitcoind.pid)
```